### ISSUE TITLE
[Enhancement]: Add a step to accept matched data before updating the book

### ISSUE THREAD

**Issue by raviwarrier** [advplyr/audiobookshelf#841]
### Describe the feature/enhancement

Currently, if I click on "quick match" from the edit modal, it will search for metadata and update the item automatically. This leaves the possibility of having wrong/incorrect/un-preferred metadata stored.
Update the item/book only when the user presses Save/Save & Close button and not automatically, thus allowing the user to revert to the original metadata by exiting out of the modal if necessary.

**Comment by stefanop1** [comment]
Maybe, a better solution could be: update the book if only one match is found, otherwise, present the list to choose the correct one

**Comment by raviwarrier** [comment]
Even then I'd want to be able to confirm or revert.

**Comment by stefanop1** [comment]
Personally, I really like the immediacy of the quick match. A 'revert' button may be useful but only as a last resort. If I don't press it, the changes will be permanent

**Comment by raviwarrier** [comment]
Something that allows the user to verify and cancel the updation if needed. Confirm button or revert button, both achieve the goal.

**Comment by wtanksleyjr** [comment]
I agree with the enhancement request that quick match not save the results (wait for the user to click "Save" or "Save and Close"); the rest of the discussion here is already implemented in the regular "Match" tab, I see little value in making the two actions do the same thing.

**Comment by raviwarrier** [comment]
I am adding to this issue again as I faced this problem and don't know how to go back to the original book details without deleting and re-adding it to my library.

Here's a book that I just added. As you can see from the file name, it is "The Relentless Revolution" by Joyce Appleby.

![Image](https://github.com/user-attachments/assets/370ade40-5a7d-454f-8224-e8dcd4924609)

Here's the MP3Tag screenshot of the same file:

![Image](https://github.com/user-attachments/assets/ba9856d2-0707-425d-8484-4fc3fe6dc175)

but when I did a quick match, it found a different book with non-related title or author, but with a similar sub-title:

![Image](https://github.com/user-attachments/assets/37e5ea89-4183-4345-aadd-8841fbb64764)

---

I deleted the book from the database (not from the physical disk) and then ran a scan again, hoping to get the original title and cover back, but it just automatically added it with the wrong title:

![Image](https://github.com/user-attachments/assets/eb48f556-02e5-457a-826e-f2cb7d43e38a)

There really needs to be a step in between "Quick Match"/"Scan" and "Save". Or at least give a way to "undo" the quick scan or match output.

I created this ticket almost 3 years ago and I think this need to go into the roadmap soon. Please.
